### ISSUE TITLE
IonClientImplTest is unstable

### ISSUE THREAD

**Issue by algattik** [eclipse-dataspaceconnector/DataSpaceConnector#786]
Observed one occurrence in CI

```
2022-03-02T07:53:37.8029611Z 
2022-03-02T07:53:37.8030022Z IonClientImplTest > resolve() FAILED
2022-03-02T07:53:37.8031340Z     org.eclipse.dataspaceconnector.ion.IonRequestException: org.eclipse.dataspaceconnector.ion.IonRequestException: Resolving the DID URI was unsuccessful: code = 404 content= {"requestId":"b3a876c99805e77eb12775e8b255678a","date":"Wed, 02 Mar 2022 07:53:37 GMT","mscv":"F7mWIGRf2VJqITc/.1","error":{"code":"discovery_service.not_found","message":"discovery_service.not_found"}}
2022-03-02T07:53:37.8032402Z         at app//org.eclipse.dataspaceconnector.ion.IonClientImpl.resolve(IonClientImpl.java:111)
2022-03-02T07:53:37.8033424Z         at app//org.eclipse.dataspaceconnector.ion.IonClientImplTest.resolve(IonClientImplTest.java:32)
2022-03-02T07:53:37.8034070Z 
2022-03-02T07:53:37.8034294Z         Caused by:
2022-03-02T07:53:37.8034889Z         org.eclipse.dataspaceconnector.ion.IonRequestException: Resolving the DID URI was unsuccessful: code = 404 content= {"requestId":"b3a876c99805e77eb12775e8b255678a","date":"Wed, 02 Mar 2022 07:53:37 GMT","mscv":"F7mWIGRf2VJqITc/.1","error":{"code":"discovery_service.not_found","message":"discovery_service.not_found"}}
2022-03-02T07:53:37.8035660Z             at app//org.eclipse.dataspaceconnector.ion.IonClientImpl.resolve(IonClientImpl.java:109)
2022-03-02T07:53:37.8036441Z             ... 1 more
2022-03-02T07:53:39.1018258Z 

```

**Comment by ndr-brt** [comment]
Will be removed (hopefully soon) by #664

**Comment by jimmarino** [comment]
Merged #664
